post-processor/amazon-import/post-processor.go

```diff
@@ -63,6 +64,10 @@ func (p *PostProcessor) Configure(raws ...interface{}) error {
 		p.config.S3Key = "packer-import-{{timestamp}}.ova"
 	}
 
+	if p.config.LicenseType == "" {
+		p.config.LicenseType = "BYOL"
```

Comment:
is byol the default license type?

that is, if no license type is configured, are images automatically imported as BYOL? I'm trying to understand if this changes the behavior at all.